data/src/main/java/com/linkedin/data/codec/AbstractJacksonDataCodec.java

```diff
@@ -133,7 +133,18 @@ protected void writeObject(Object object, JsonGenerator generator) throws IOExce
 
   protected Data.TraverseCallback createTraverseCallback(JsonGenerator generator)
   {
-    return new JacksonTraverseCallback(generator);
+    return createTraverseCallback(generator, false);
+  }
+
+  /**
+   * Create {@link JacksonTraverseCallback} instance for Data object traverse
```

Comment:
JacksonTraverseCallback -> Traverse.Callback
Avoid implementation details in the java doc

Reply:
I wouldn't say I totally agree with you in this one. But I made the change according to your suggestion/

The reason is that this method take in "generator" as input parameter in order to return Data.TraverseCallback, this is already very implementation specific and very detailed implementation description. Something unavoidable.